src/utils/configurationDataHandler.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+export const handleEtrustedInteraction = (
+  token: string | undefined,
+  allState: any,
+  interactionType: string,
+  callback: (token: string, payload: any) => Promise<void>
```

Comment:
If you don't mind about the payload type then it's always better to use `unknown` and you don't need to disable eslint rule ;)

Reply:
I Changed the type to unknown, but the rule of disabling should stay there because they configured the eslint to don't allow console